core/src/test/java/io/fabric8/maven/core/handler/ContainerHandlerTest.java

```diff
@@ -257,6 +257,23 @@ public void getImageNameTest(){
         assertNull(containers.get(3).getImage());
     }
 
+    @Test
+    public void getRegistryTest() {
+        ContainerHandler handler = new ContainerHandler(project1, envVarHandler, probeHandler);
```

Comment:
sure, not an issue of this test, but I really wonder why this test class keeps all setup in member variables and not in local variables (e.g. a local MavenProject() instead of the member var). As it is now every test needs to be very careful to cleanup afterwards, as all the test method share state. It could be also that the tests become flaky as test methods are running in different order. 

Actually I highly recommend to use no shared state between test methods, except when you need extra features like injected mocks.

Reply:
@piyush1594 any particular reason why you introduced project, project1, project2 as member variables and don't create them as local variables when you need them ? It makes maintaining such tests quite hard.